src/conv.rs

```diff
@@ -0,0 +1,131 @@
+//! Implements internal type conversion utilities.
+
+use web3::types::U256;
+
+/// Lossy conversion from a `U256` to a `f64`.
+pub fn u256_to_f64(value: U256) -> f64 {
```

Comment:
I'm surprised we have to implement this kind of logic by hand. Can you explain why you are using 54 significant bits? 

Wouldn't converting to and from string be easier?

Reply:
Cool, maybe this is something we can try to merge upstream (can keep it in here for now)

Reply:
Created #141